src/models/Plan.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const schema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  createdAt: {
+    type: Date,
+  },
+  places: [
+    {
+      id: {
```

Comment:
rename this to be `googlePlaceId` which clearly indicates that it is the google place id and not some internal id